src/mod/resources/connectormod.mixins.json

```diff
@@ -15,7 +15,10 @@
   ],
   "client": [
     "MinecraftMixin",
-    "ItemBlockRenderTypesMixin"
+    "ItemBlockRenderTypesMixin",
+    "RecipeBookManagerMixin",
+    "RecipeBookCategoriesMixin",
+    "RecipeBookCategoriesAccessor"
```

Comment:
Can we move these to their own `recipebook` subpackage to better keep track of the patch's changes?